chain/cosmos/tx.go

```diff
@@ -3,9 +3,14 @@ package cosmos
 import (
 	"context"
 	"crypto/sha256"
+	"encoding/hex"
 	"fmt"
 	"math/big"
 
+	"github.com/renproject/multichain"
+
+	"github.com/cosmos/cosmos-sdk/x/auth/tx"
+
 	"github.com/btcsuite/btcd/btcec"
 
 	"github.com/cosmos/cosmos-sdk/client"
```

Comment:
Possible to format these? i.e. remove the spaces between them

Reply:
I did goimport and gofmt not sure why it didn't change this one. Will try to manually change it

Reply:
I think they need to be grouped together for the auto formatting to apply